mewarpx/mewarpx/plotting.py

```diff
@@ -96,8 +94,6 @@ class ArrayPlot(object):
             'legendsize': 'medium',
             'titlesize': 'large',
             'cmap': 'viridis',
```

Comment:
Let's make 'viridis' the default on line 43 (I can't comment there for some reason in my current view). Then eliminate everywhere else.
```suggestion
```